### DISCUSSION TITLE
Backend Ideas

### DISCUSSION THREAD

**Discussion by diegoq48** [uprm-inso4101-2023-2024-s2/semester-project-personalfinancemanager#4] [Ideas]
Enter any suggestion for technologies or strategies for the projects backend

**Comment by Rodal1** [comment]
I've previously used phpMyadmin together with MySQL to handle data for a website that I linked with Piwigo (Image Database), we could take an approach using a VPS.

Here an example of phpMyAdmin:
![image](https://github.com/uprm-inso4101-2023-2024-s2/semester-project-personalfinancemanager/assets/90566705/3fddb11f-7d16-4ab8-bcda-aca2abe1010c)

Summarized, phpMyAdmin handles MySQL databases through web, this could be implemented on a private server(VPS), for the backend to be worked easier.

**Reply by Rodal1** [reply]
And it can run the web app on Apache or Nginx at the same time. (I've worked with that before)

**Reply by diegoq48** [reply]
Its a good idea pero a little to late theres a baseplate in cloudflare workers for the db if you can raise some points against it lmk.
